pkg/networkmanager/interfaces.js

```diff
@@ -143,7 +143,7 @@ function NetworkManagerModel() {
     var self = this;
     var byteorder = null;
 
-    var client = cockpit.dbus("org.freedesktop.NetworkManager");
+    var client = cockpit.dbus("org.freedesktop.NetworkManager", { superuser: true });
```

Comment:
So our goal is still to prefer that a service does its own access control (via polkit or other means) rather than us just becoming superuser, right?